test/functional/data/invalid_txs.py

```diff
@@ -49,7 +49,10 @@
     OP_LSHIFT,
     OP_RSHIFT
 )
-basic_p2sh = sc.CScript([sc.OP_HASH160, sc.hash160(sc.CScript([sc.OP_0])), sc.OP_EQUAL])
+from test_framework.script_util import (
+    script_to_p2sh_script,
+)
+basic_p2sh = script_to_p2sh_script(sc.CScript([sc.OP_0]))
```

Comment:
`CScript` is already imported. So just  `script_to_p2sh_script(CScript([sc.OP_0]))`. 

So can be done for `OP_0` if its imported in the list above.

Reply:
Thanks for reviewing! Good point, got rid now of the redundant `import script as sc` and extended the list of OP_... imports accordingly, took also the chance to sort them. Should Look cleaner now.

Reply:
Looks good now.. Thanks for considering.